newfile-featrure.txt

```diff
@@ -0,0 +1 @@
+hellow world
```

Comment:
The name of the file should have been 'feature.txt'.
This file should have been empty at the first commit.